vulnerabilities/tests/test_api.py

```diff
@@ -121,8 +121,7 @@ def test_ubuntu_response(self):
         self.assertEqual(1, len(result['vulnerabilities']))
 
         vuln = result['vulnerabilities'][0]
-        self.assertEqual(1, len(vuln['references']))
-        self.assertEqual('CVE-2012-3386', vuln['references'][0]['reference_id'])
+        self.assertEqual(0, len(vuln['references']))
```

Comment:
Note: if we decide to use pytest and its dnajo extension, we could start using plain asserts everywhere.

Reply:
Right, I had that thought as well. Wrote it down in a ticket now: https://github.com/nexB/vulnerablecode/issues/127